common/constants/default.go

```diff
@@ -45,8 +45,6 @@ const (
 	DefaultRemoteRuntimeEndpoint       = "unix:///var/run/dockershim.sock"
 	DefaultRemoteImageEndpoint         = "unix:///var/run/dockershim.sock"
 	DefaultPodSandboxImage             = "kubeedge/pause:3.1"
```

Comment:
```
# uname -i
aarch64

# docker image inspect kubeedge/pause:3.1 | grep "Architecture"
        "Architecture": "amd64",
```

we need to push the multi-arch docker image to docker hub first. @fisherxu  who has the permission to update this docker image?
https://hub.docker.com/r/kubeedge/pause/tags?page=1&ordering=last_updated